.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -125,7 +125,7 @@ def FalsePred(_):
 def TruePred(_):
     return True
 
-_VC2019 = VcSpec(2019)
+_VC2019 = VcSpec(2019, ["14", "27"], hide_version=True)
```

Comment:
Although it sounds perfect, it is impractical that you may install MSVC toolchain with a different revision number.

Reply:
Also I don't think it make sense to limit the toolchain with a revision number. Normally it won't break anything when upgrading AA.BB.CC to AA.BB.DD. Even if it happens, there's no way we could workaround it (except downgrading it again, but that's a different story) since the macro `_MSC_VER` only reflects the major version and the minor version. (e.g. `1928` -> `14.27`)